_layouts/product.html

```diff
@@ -183,6 +183,16 @@
   <td {% if diff <= 0 %} class = "txt-linethrough" {% endif %} >      <!-- if the support finished add txt-linethrough class -->
     {% if releaseLink != "" %}
       <a href="{{releaseLink}}" title="Release Notes / Changelog">{{latestVersionNumber}}</a>
+      {% if  diff > 1 %}
+      &ensp;
+      <a href="http://www.google.com/calendar/render?action=TEMPLATE
+&text=[endoflife.date] {{page.title}} {{r.releaseCycle}} EOL reached
+&dates={{r.eol | replace:'-',''}}T000000/{{r.eol | replace:'-',''}}T000000
+&details=It will be end of life for {{page.title}} {{r.releaseCycle}} . You can visit https://endoflife.date{{page.permalink}} to get more information.
```

Comment:
I think it is safer to use `{{ page.url | absolute_url }}` instead of `https://endoflife.date{{page.permalink}}`.

And may be this message would be better in this case : `{{page.title}} {{r.releaseCycle}} will reach its end of life. For more information, see {{ page.url | absolute_url }}.`